docs/guides/node-operator-manual.md

```diff
@@ -276,3 +326,43 @@ docker run --rm -it \
   --datadir /root/.lighthouse/data \
   --directory /root/validator_keys
 ```
+
+### Checking the keys of all Lido Node Operators
+
+After keys are submitted, it's critical to first wait for the key submission transactions to be mined and after that check all Lido keys for validity.
```

Comment:
The tooling for the key checking works only with the on-chain data. Make sure to get the key submission transactions confirmed before checking the keys.